src/strategies/rust.ts

```diff
@@ -101,6 +101,18 @@ export class Rust extends BaseStrategy {
       });
     } else {
       const manifestPath = this.addPath('Cargo.toml');
+      const manifestContent = await this.getContent(manifestPath);
+      if (!manifestContent) {
+        logger.warn('unable to find Cargo.toml');
+        return updates;
+      }
+      const manifest = parseCargoManifest(manifestContent.parsedContent);
+      if (manifest.package?.name) {
+        versionsMap.set(manifest.package.name, version);
+      } else {
+        logger.warn('No crate package name found');
+      }
+
```

Comment:
Will `this.getDefaultPackageName()` work for this?

Reply:
Yes! Much better! 😄